include/config/test.h

```diff
@@ -10,4 +10,1125 @@
 #undef B_EXPANDED_TYPE_NAMES
 #define B_EXPANDED_TYPE_NAMES TRUE
 
+#undef P_MEGA_EVOLUTIONS
+#define P_MEGA_EVOLUTIONS                TRUE
+#undef P_PRIMAL_REVERSIONS
+#define P_PRIMAL_REVERSIONS              TRUE
+#undef P_ULTRA_BURST_FORMS
+#define P_ULTRA_BURST_FORMS              TRUE
+#undef P_GIGANTAMAX_FORMS
+#define P_GIGANTAMAX_FORMS               TRUE
+#undef P_TERA_FORMS
+#define P_TERA_FORMS                     TRUE
+#undef P_FUSION_FORMS
+#define P_FUSION_FORMS                   TRUE
+#undef P_REGIONAL_FORMS
+#define P_REGIONAL_FORMS                 TRUE
+#undef P_ALOLAN_FORMS
```

Comment:
we don't need the ones for stuff like `P_REGIONAL_FORMS` if they're only used as an easy way to disable multiple other defines
```suggestion
#define P_FUSION_FORMS                   TRUE
#undef P_ALOLAN_FORMS
```